README.md

```diff
@@ -235,6 +235,30 @@ rainbow table attacks. There are multiple ways to do so, listed by order of prec
 
 The salt should be of the same encoding as the associated data to hash.
 
+### Sanitize hash
+
+> _Support: introduced in version 1.6.0_
+
+You can sanitize a hash before creation and querying. This might be useful in case you would like to find a User with the name of `François ` with a query input of `francois`.
+
+There are several sanitize options:
+
+```
+/// @encryption:hash(email)?sanitize=lowercase  <- lowercase hash
+/// @encryption:hash(email)?sanitize=uppercase  <- uppercase hash
+/// @encryption:hash(email)?sanitize=trim       <- trim start and end of hash
+/// @encryption:hash(email)?sanitize=spaces     <- remove spaces in hash
+/// @encryption:hash(email)?sanitize=diacritics <- remove diacritics like ç or é in hash
```

Comment:
**suggestion**: Based on the implementation, `normalize` could be a more appropriate name for what happens under the hood. Also, TIL about the [`normalize`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize) function, thanks!

Reply:
Hah, I actually meant the value of `diacritics`, not the whole `sanitize` key name, but I guess it works with `normalize` as well, let's keep that.

Reply:
Oh in that case I would keep it to `diacritics`, because the word `normalize` has a wider meaning to me then just the diacritics conversion of a string. Who knows what kind of other "normalizations" you would like to add in the future.